funnel/forms/project.py

```diff
@@ -165,12 +165,11 @@ class CfpForm(forms.Form):
     cfp_end_at = forms.DateTimeField(
         __("Proposal submissions close at"),
         validators=[
+            forms.validators.Optional(),
             forms.validators.AllowedIf(
                 'cfp_start_at',
                 message=__("This requires open time for submissions to be specified"),
             ),
-            forms.validators.RequiredIf('cfp_start_at'),
-            forms.validators.Optional(),
```

Comment:
We no longer require cfp end date? Open ended?

Reply:
Open-ended CfPs were in the spec, for round-the-year projects. Requiring an end-date was a bug. Fixed now.